pages/playground.js

```diff
@@ -6,9 +6,10 @@ import Button from "@components/Button";
 import Modal from "@components/Modal";
 import Input from "@components/form/Input";
 import UserPage from "@components/user/UserPage";
+import env from '@config/env'
 
 export default function Playground() {
-  const BASE_URL = process.env.NEXT_PUBLIC_BASE_URL;
+  const BASE_URL = env.NEXT_PUBLIC_BASE_URL;

```

Comment:
Can't change this as it is executed on the fron-end

Reply:
Didn't get you. What is the difference when this gets executed on the front end? Ultimately, the file will be loaded right?

Reply:
The logger code will not work on the front end